packages/Modal/src/components/FocusLock/FocusLock.js

```diff
@@ -8,7 +8,18 @@ const defaultProps = {
   returnFocus: true,
 };
 
-const FocusLock = props => <FocusLockLibrary {...props} />;
+const FocusLock = props => {
+  // ignore any focusable elements in pendo container
+  function whiteList(node) {
+    const pendoContainer = document.getElementById("pendo-base");
+
+    if (!pendoContainer) return true;
+
+    return !pendoContainer.contains(node);
```

Comment:
Would something like this work, and also allow consumers to pass a custom `whiteList` prop?

```js
    return !pendoContainer.contains(node) || props.whiteList(node);
```

Reply:
That will return `true` as long as `props.whiteList` returns `true`, is that what we expect? If that's correct, consumers can just override the whitelist rule by passing their `whitelist` function through props. We don't need to do extra work

Reply:
I think we should do this if we want to filter out both pendo and user specified elements:
```jsx
    const { whiteList: whiteListProp } = props;
    const pendoContainer = document.getElementById("pendo-base");
    const whiteListPropResult = whiteListProp ? whiteListProp(node) : true;

    if (!whiteListPropResult) return false;

    if (!pendoContainer) return whiteListPropResult;

    return !pendoContainer.contains(node);
```

Reply:
I think we should do this if we want to filter out both pendo and user specified elements:
  ```jsx
       const { whiteList: whiteListProp } = props;
       const pendoContainer = document.getElementById("pendo-base");
       const whiteListPropResult = whiteListProp ? whiteListProp(node) : true;

       if (!whiteListPropResult) return false;

       if (!pendoContainer) return true;

       return !pendoContainer.contains(node);
  ```